amr-wind/ocean_waves/boundary_ops/OceanWavesBoundary.cpp

```diff
@@ -238,6 +264,10 @@ void OceanWavesBoundary::set_inflow_sibling_velocity(
 
         const int idir = ori.coordDir();
         const auto& domain_box = geom.Domain();
+
+        amrex::IntVect shift_to_interior = {0, 0, 0};
```

Comment:
no that this matters much but just in case, you can use `TheDimensionVector` or something like that to get a 1 in the `dir` location. You would still need to negate it after to get the `-1` with something like `shift_to_interior *= -1` if ori is low.